packages/angular/projects/ui-angular/src/lib/components/authenticator/components/form-field/form-field.component.ts

```diff
@@ -58,7 +58,7 @@ export class FormFieldComponent implements OnInit {
       this.authenticator.authState
     );
     const { validationError } = formContext;
-    return translate(validationError[this.name]);
+    return translate(validationError[this.name] as string);
```

Comment:
It's because we changed the type of `ValidationError` to include `string | string[]`. So TS gets confused, and you have to use type inference to fix it.

A long term solution will to change `ValidationError` to be `string[]`, but that will be a breaking change. As discussed in an earlier comment .